### PR TITLE
Various entity mappings

### PR DESCRIPTION
**Pull request by YanisBft**
Renamed some classes, methods and fields to make them consistent

### CODE REVIEW

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/entity/EnderEyeEntity.mapping [hunk 1]
```diff
@@ -1 +1,2 @@
 CLASS asi net/minecraft/entity/EnderEyeEntity
+	FIELD f uses I
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/EnderEyeEntity.mapping]: useCount? "uses" is a bit weird here to me IMO

File: mappings/net/minecraft/entity/decoration/AbstractDecorationEntity.mapping [hunk 2]
```diff
@@ -6,8 +6,9 @@ CLASS aph net/minecraft/entity/decoration/AbstractDecorationEntity
 		ARG 1 type
 	METHOD <init> (Lahq;Lbdf;Les;)V
 		ARG 1 world
-	METHOD a copyEntityData (Lahm;)V
+	METHOD a getBreakEvents (Lahm;)V
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/decoration/AbstractDecorationEntity.mapping]: get... and it returns void?

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/decoration/AbstractDecorationEntity.mapping (hunk 2 above)
Comment (YanisBft) [reply mappings/net/minecraft/entity/decoration/AbstractDecorationEntity.mapping]: Maybe fireBreakEvents/firePlaceEvents could be better?

Comment (asiekierka) [reply mappings/net/minecraft/entity/decoration/AbstractDecorationEntity.mapping]: Better, but it still sounds very off. Why not just onBreak/onPlace?

File: mappings/net/minecraft/entity/mob/ShulkerEntity.mapping [hunk 3]
```diff
@@ -9,11 +10,13 @@ CLASS aqv net/minecraft/entity/mob/ShulkerEntity
...
 	FIELD bB ATTR_COVERED_ARMOR_BONUS Laim;
+	FIELD bz COLOR Lpx;
 	FIELD b ATTACHED_FACE Lpx;
 	FIELD c ATTACHED_BLOCK Lpx;
 	FIELD d PEEK_AMOUNT Lpx;
 	METHOD a setPeekAmount (I)V
 	METHOD dM getAttachedFace ()Lex;
 	METHOD dN getAttachedBlock ()Les;
 	METHOD dO getPeekAmount ()I
+	METHOD dT getColor Lawr;
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/mob/ShulkerEntity.mapping]: Shouldn't that be ()Lawr;? How did you make those mappings, exactly? Enigma or a third-party tool?

File: mappings/net/minecraft/entity/mob/SlimeEntity.mapping [hunk 4]
```diff
@@ -3,4 +3,10 @@ CLASS aqy net/minecraft/entity/mob/SlimeEntity
 		FIELD k slime Laqy;
 	FIELD bA onGroundLastTick Z
 	FIELD bz SLIME_SIZE Lpx;
+	FIELD b sizeX F
+	FIELD c sizeY F
+	FIELD d sizeZ F
+	METHOD dO getSquishSound Lxy;
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/mob/SlimeEntity.mapping]: Same here... and down below...

File: mappings/net/minecraft/entity/projectile/AbstractFireballEntity.mapping [hunk 5]
```diff
@@ -0,0 +1,2 @@
+CLASS asj net/minecraft/entity/projectile/AbstractFireballEntity
+	FIELD f itemStack Lpx;
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/projectile/AbstractFireballEntity.mapping]: Just "stack" is fine.

File: mappings/net/minecraft/entity/projectile/ArrowEntity.mapping [hunk 6]
```diff
@@ -2,6 +2,7 @@ CLASS asf net/minecraft/entity/projectile/ArrowEntity
 	FIELD aq COLOR Lpx;
 	FIELD ar potion Lazo;
 	FIELD as effects Ljava/util/Set;
+	FIELD at hasColor Z
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/projectile/ArrowEntity.mapping]: hasColor for the method; use something like "colorSet".

**Review by modmuss50 — APPROVED** [review]

**Pull request closed without merging**